app/views/role_change_mailer/notify_role_end.erb

```diff
@@ -1,4 +1,4 @@
-<p>The role has been removed for <%= @role.user.name %> in <%= @group_type_name %> on <%= @role.end_date %>.</p>
+<p>The role has been removed for <%= UserRole.user(@role).name %> in <%= @group_type_name %> on <%= @role[:end_date] %>.</p>
```

Comment:
You're changing this to `@role[:end_date]` which looks like a "backported legacy role". Will this notation work with actual new roles?

Reply:
Yes, this works for actual new roles as well, I confirmed. Attaching the screenshot:

<img width="968" alt="Screenshot 2024-03-09 at 16 25 28" src="https://github.com/thewca/worldcubeassociation.org/assets/9512698/2af9cd4b-b02a-4221-b68e-7478e5533b74">